src/typings/typeof-is.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'typeof-is';
```

Comment:
vervelend dat hier geen `@types` voor zijn, maar deze module lijkt mij klein genoeg om zelf gewoon te typen.
wat je nu doet is 'm als any declareren namelijk.

Reply:
Ik heb die package er maar uit gehaald en zelf een kleine service geschreven. Was een beetje een kansloze package.